test/test_files/abl_wallrans_neutral/abl_wallrans_neutral.inp

```diff
@@ -0,0 +1,109 @@
+# MLMG
+mac_proj.num_pre_smooth 			 = 32
+mac_proj.num_post_smooth 			 = 32
+mac_proj.mg_rtol 			 = 1.0e-4
+mac_proj.mg_atol 			 = 1.0e-6
+mac_proj.maxiter 			 = 360
+diffusion.mg_rtol                        = 1e-4
+diffusion.mg_atol                        = 1e-6
+temperature_diffusion.mg_atol            = 1e-6
+temperature_diffusion.mg_rtol            = 1e-4
+tke_diffusion.mg_rtol            = 1e-6
+tke_diffusion.mg_atol            = 1e-6
+nodal_proj.num_pre_smooth 			 = 32
+nodal_proj.num_post_smooth 			 = 32
+nodal_proj.mg_rtol 			 = 1.0e-4
+nodal_proj.mg_atol 			 = 1.0e-6
+nodal_proj.maxiter 			 = 360
+
+
+ABL.initial_wind_profile      = true
+
+ABL.temperature_heights = 0  4.01569  8.03137  12.0471  16.0627  20.0784  24.0941  28.1098  32.1255  36.1412  40.1569  44.1725  48.1882  52.2039  56.2196  60.2353  64.251  68.2667  72.2824  76.298  80.3137  84.3294  88.3451  92.3608  96.3765  100.392  104.408  108.424  112.439  116.455  120.471  124.486  128.502  132.518  136.533  140.549  144.565  148.58  152.596  156.612  160.627  164.643  168.659  172.675  176.69  180.706  184.722  188.737  192.753  196.769  200.784  204.8  208.816  212.831  216.847  220.863  224.878  228.894  232.91  236.925  240.941  244.957  248.973  252.988  257.004  261.02  265.035  269.051  273.067  277.082  281.098  285.114  289.129  293.145  297.161  301.176  305.192  309.208  313.224  317.239  321.255  325.271  329.286  333.302  337.318  341.333  345.349  349.365  353.38  357.396  361.412  365.427  369.443  373.459  377.475  381.49  385.506  389.522  393.537  397.553  401.569  405.584  409.6  413.616  417.631  421.647  425.663  429.678  433.694  437.71  441.725  445.741  449.757  453.773  457.788  461.804  465.82  469.835  473.851  477.867  481.882  485.898  489.914  493.929  497.945  501.961  505.976  509.992  514.008  518.024  522.039  526.055  530.071  534.086  538.102  542.118  546.133  550.149  554.165  558.18  562.196  566.212  570.227  574.243  578.259  582.275  586.29  590.306  594.322  598.337  602.353  606.369  610.384  614.4  618.416  622.431  626.447  630.463  634.478  638.494  642.51  646.525  650.541  654.557  658.573  662.588  666.604  670.62  674.635  678.651  682.667  686.682  690.698  694.714  698.729  702.745  706.761  710.776  714.792  718.808  722.824  726.839  730.855  734.871  738.886  742.902  746.918  750.933  754.949  758.965  762.98  766.996  771.012  775.027  779.043  783.059  787.075  791.09  795.106  799.122  803.137  807.153  811.169  815.184  819.2  823.216  827.231  831.247  835.263  839.278  843.294  847.31  851.325  855.341  859.357  863.373  867.388  871.404  875.42  879.435  883.451  887.467  891.482  895.498  899.514  903.529  907.545  911.561  915.576  919.592  923.608  927.624  931.639  935.655  939.671  943.686  947.702  951.718  955.733  959.749  963.765  967.78  971.796  975.812  979.827  983.843  987.859  991.875  995.89  999.906  1003.92  1007.94  1011.95  1015.97  1019.98  1024  3048 
```

Comment:
oh this is a bummer... can we read in from a file? I know you have an issue open on this, I think now is the time to address it. There are lots of examples of reading data from file, lmk if you need a pointer.

Reply:
I can only fix it for the new changes. There are additional dependencies for ABL.temperature_XXXX that changing that part of the code will break it in other places.